Source/msg.cpp

```diff
@@ -3038,15 +2994,12 @@ size_t ParseCmd(int pnum, const TCmd *pCmd)
 	case CMD_PLRDAMAGE:
 		return OnPlayerDamage(pCmd, player);
 	case CMD_OPENDOOR:
-		return OnOpenDoor(pCmd, pnum);
 	case CMD_CLOSEDOOR:

```

Comment:
I know there was some funny business going on with the message being sent multiple times. Hope you made extra sure that doors aren't flapping as a consequence :)